packages/core/src/app/checkout/Checkout.tsx

```diff
@@ -76,6 +76,7 @@ export interface CheckoutState {
     isRedirecting: boolean;
     hasSelectedShippingOptions: boolean;
     isBuyNowCartEnabled: boolean;
+    isStripeLinkAuthenticated: boolean;
```

Comment:
This looks like an ever growing list of states? Can we find another way to add StripeLink state?

Reply:
Hey @bc-peng, I was thinking to create an useContext or create a new interface with that field and extend by the class or inserting a hidden element with that info into the dom (but I don’t like it), please let me know if any of those options would be great or If you have any other idea would be great too :D

Reply:
In `checkout-js`, we use `checkout-sdk-js` as the global state store. We can tell if checkout is in a loading state by using a sdk function. Is there a similar solution on `Stripe` end?